templates/python/custom-copilot-basic/src/requirements.txt

```diff
@@ -1,3 +1,3 @@
 python-dotenv
 aiohttp
-teams-ai~=1.2.0
\ No newline at end of file
+teams-ai~=1.4.0
```

Comment:
let's change the version to >=1.4.0 & < 2.0.0, so the template can adopt latest SDK version automatically

Reply:
sure